gzip_logs_tools/log_compressor/log_compressor.c

```diff
@@ -790,7 +796,9 @@ create_pid_file(const char *pid_file)
 	}	
 
 	sprintf(buf, "%ld\n", (long) getpid());
-	if (write(fd, buf, strlen(buf)) != strlen(buf))
+	size_t buf_len = strlen(buf);
+	ssize_t written_bytes = write(fd, buf, buf_len);
```

Comment:
please move the var definitions to the beginning of the function (next to `int fd` etc.)

Reply:
moved (I also renamed to bytes_written since it seems that't he naming convention you used elsewhere)